### DISCUSSION TITLE
blog/learning-and-inspiration

### DISCUSSION THREAD

**Discussion by giscus** [tommm2/jinrup.dev#35] [Announcements]
# blog/learning-and-inspiration

製作個人網站的學習與啟發.

https://jinrup.vercel.app/blog/learning-and-inspiration

**Comment by nelsonlaidev** [comment]
很棒，繼續努力！ 💪🏻💪🏻

**Reply by tommm2** [reply]
謝了兄弟，by the way 你的網站真的做的很漂亮
